### DISCUSSION TITLE
Binding not working on Linkind 1-key remote control

### DISCUSSION THREAD

**Discussion by Docano** [Koenkk/zigbee2mqtt#10067] [Devices]
Hello, friends.

I've been trying, unsuccessfully, to properly bind a Linkind 1-Key Remote to a Linkind Plug. Hoping someone can point me in a helpful direction because I've exhausted my troubleshooting tools.

These were purchased as a pair and initially bound to each other by the manufacturer. But after adding the plug to Z2M, the remote is no longer paired to the plug!

Adapter: [ConBee II](https://www.zigbee2mqtt.io/guide/adapters/#other)
Environment: Z2M 1.22.0-2 in Home Assistant OS on a Raspberry Pi 3b+
Product links: [Amazon](https://www.amazon.com/Linkind-Control-Wireless-Controlled-Expandable/dp/B08FDD99VD/) , [Linkind.com](https://www.linkind.com/product/1-key-remote-control-for-linkind-zigbee-smart-lights/)
Z2M link to the remote: [Linkind ZS232000178](https://www.zigbee2mqtt.io/devices/ZS232000178.html)
Z2M link to the plug: [Linkind ZS190000118](https://www.zigbee2mqtt.io/devices/ZS190000118.html)

The Z2M front-end indicates that the remote (friendly name tree_plug_remote) has been successfully bound to the plug (friendly name tree_plug). But a button press on the remote doesn't do anything. As you can see from the screenshot, I've tried binding to the plug directly and also binding to a group containing the plug. Neither works :(

![PkqlwDOD2C](https://user-images.githubusercontent.com/23645450/145042577-035c07c4-d840-4d5b-9df5-41d990c3a4f4.png)

Can anyone offer some advice on how to get the binding to work? Or troubleshooting advice? 

Do you think this an issue with the ConBee II? I don't have another Zigbee adapter, but I'd be willing to try another.

The simple workaround would be to create an automation in homeassistant, but my HassOS instance has been unstable lately, and I need this remote to work even if the Mosquitto broker goes offline.

Thank you!

**Comment by kannszzz** [comment]
Did you ever figure out binding for linkind plug and button?
